style.css

```diff
@@ -241,3 +241,64 @@ p {
 .green {
   color: #36b37e;
 }
+
+.contactContainer {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 24px;
+}
+
+.contactForm {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  margin-top: 14.8px;
+}
+
+.singleLineInput {
+  box-sizing: border-box;
+  width: 327px;
+  height: 48px;
+  padding: 15px 16px;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+}
+
+.singleLineInput::placeholder {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.commentArea {
+  box-sizing: border-box;
+  padding: 12px;
+  width: 327px;
+  height: 114px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+}
+
+.contactForm button {
+  margin-bottom: 28px;
+}
```

Comment:
- [ ] I think it would be better too to add some button interactions such as hover, active (when it's pressed), disabled, and enable (default state) as shown in the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27).

Reply:
@Debas-31 Thank you for the review. I have a question since this has already been added but you can only press on it when all the required fields have been filled. Should it also work regardless of that? 
![image](https://user-images.githubusercontent.com/114885671/199364375-8ec67c7a-7e34-41bd-a709-f67c2ac3fb68.png)
The button is pressed with all the required fields.
This is the code where it was added before and this button inherits the behavior.
![image](https://user-images.githubusercontent.com/114885671/199364551-c4caebb1-12c3-4da9-a1aa-a637e6749dce.png)